src/carRace/executionProcess.js

```diff
@@ -0,0 +1,18 @@
+import { movingForward } from "./movingForward.js";
+import { CAR_RACE } from "../constants/constants.js";
+
+export function executionProcess(players, attemptsNumber) {
+  const playersArray = players.split(",");
+  const playersObject = new Object();
+
+  playersArray.forEach((player) => {
+    playersObject[`${player}`] = "";
+  });
```

Comment:
```suggestion
playersObject[player] = "";
```
이렇게도 computed된 key를 사용할 수 있지 않을까 싶어요!

Reply:
앗 왜저렇게 했지..!! 감사합니다 ㅎㅎ 더 꼼꼼히 보고 제출해볼게요! ㅎㅎ